radify/angular-scaffold
# Add unit tests and documentation

* Add documentation README.md which explains angular-scaffold's usage
* Add sample project illustrating basic usage of angular-scaffold
  * Includes a very simple Express API so users can get up and running immediately
* Add package.json
  * Prepares the project for NPM
  * Supplies dependencies for the unit tests
* Get the unit tests running with Gulp
* Upgrade unit tests to Jasmine 2.0

Test script
--

* Go to https://github.com/radify/angular-scaffold/tree/docs
* Read the documentation
  * **Assert that** it makes sense
* Clone the project and `git checkout docs`
  * Install and run the sample API as per [the documentation](https://github.com/radify/angular-scaffold/tree/docs)
  * **Assert that** you can run the sample API and it works as you would expect

Time spent
--

8 hours
